core/input/fixedkeys.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * noVNC: HTML5 VNC client
+ * Copyright (C) 2017 Pierre Ossman for Cendio AB
+ * Licensed under MPL 2.0 or any later version (see LICENSE.txt)
+ */
+
+/* [module]
+ * import KeyTable from "./keysym";
+ */
+
+"use strict";
+
+var fixedkeys = {
```

Comment:
If it's huge then it's probably not the same thing. This is **not** a mapping between DOM key codes and keysyms as such a thing would in most cases be incorrect. This is merely the subset of keys that we assume are mapped the same on every layout.

I can add a comment to this file as well.